### PR TITLE
feat: add container entity in explore page

### LINKED ISSUE #10616: UI: Add Container entity on explore page.
We have added a new service Object store which has a container as an entity, [this](https://github.com/open-metadata/OpenMetadata/pull/10562) PR has added support for the search for the same.

On UI we have to add a new tab for the `Containers` entity.

### PR DESCRIPTION
**Pull request by karanh37**
### Describe your changes :

I worked on the ..... because ...
Added container tab in Explore page.
Closes #10616 

#
### Type of change :

- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

Screenshot 2023-03-16 at 8 30 11 PM
Screenshot 2023-03-17 at 5 52 56 PM

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers

Frontend: @open-metadata/ui

### CODE REVIEW

**Comment by karanh37** [comment]
Screenshot 2023-03-16 at 8 30 11 PM

File: openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/ContainerSummary/ContainerSummary.component.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import SVGIcons from 'utils/SvgUtils';
+import SummaryList from '../SummaryList/SummaryList.component';
+import { BasicEntityInfo } from '../SummaryList/SummaryList.interface';
+
+interface ContainerSummaryProps {
+  entityDetails: Container;
+  componentType?: string;
+  tags?: TagLabel[];
+  isLoading?: boolean;
+}
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/ContainerSummary/ContainerSummary.component.tsx:38]: move it to interface

**Review by chirag-madlani — COMMENTED** [review]
Please add tests for `openmetadata-ui/src/main/resources/ui/src/components/Explore/EntitySummaryPanel/ContainerSummary/ContainerSummary.component.tsx`

**Review by harshach — APPROVED** [review]
UX wise LGTM

**Pull request merged**